docs/application-connector/docs/010-details-ac-deployment.md

```diff
@@ -19,16 +21,18 @@ You can override the following parameters:
 - **sourceType** is the Event source type
 - **sourceNamespace** is the organization that publishes the Event
 
-### Install an Application Connector on the cluster
+Follow the **Set up a Remote Environment on Minikube** getting started guide to learn more about installing and setting up a Remote Environment on
+a local Kyma installation.
+
+## Install a Remote Environment on a cluster Kyma deployment
```

Comment:
Deployment?

Reply:
This is just a synonym for "installation", shouldn't be capitalized.